clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -440,10 +460,26 @@ export const {
   useCreateStorageMutation,
   useCreateStorageSecretsMutation,
   useCreateConfigurationSettingsMutation,
+  useGetConfigurationSettingsQuery,
   useGetMessagingConfigurationDetailsQuery,
   useGetActiveMessagingProviderQuery,
   useGetActiveStorageQuery,
   useCreateMessagingConfigurationMutation,
   useCreateMessagingConfigurationSecretsMutation,
   useCreateTestConnectionMessageMutation,
 } = privacyRequestApi;
+
+const EMPTY_CORS_DOMAINS = { domains: [] } as { domains: string[] };
```

Comment:
I think something like this would be a little more strict (`as` does a cast, whereas `:` is defining it)

```suggestion
const EMPTY_CORS_DOMAINS : { domains: string[] } = { domains: [] };
```

Reply:
I ended up finding a pretty clean typing solution for this line that's different from the suggestion. I'm going to resolve this comment. Let me know what you think of the new version